fullstop-plugins/fullstop-ami-plugin/src/test/java/org/zalando/stups/fullstop/plugin/ami/config/AmiPluginAutoConfigurationTest.java

```diff
@@ -4,20 +4,20 @@
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
-import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 import org.zalando.stups.fullstop.plugin.EC2InstanceContextProvider;
 import org.zalando.stups.fullstop.plugin.ami.AmiPlugin;
 import org.zalando.stups.fullstop.violation.ViolationSink;
 
-import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
+import static org.assertj.core.api.Assertions.assertThat;
 
 
 @RunWith(SpringJUnit4ClassRunner.class)
```

Comment:
There is an alias class for `SpringJUnit4ClassRunner` called `SpringRunner` now.